copter/source/docs/channel-7-and-8-options.rst

```diff
@@ -233,7 +233,17 @@ Reset :ref:`Auto <auto-mode>` to run the first mission command in the command li
    </td>
    </tr>
    <tr>
-   <td><strong>Retract Mount</strong></td>
+   <td><strong>Retract Mount1</strong></td>
```

Comment:
no need to make these changes...its a deprecated page and they are not updated

Reply:
@Hwurzburg san.  I understand. I have reverted the change.